### PR TITLE
Add documentation on CPU models for the bootcamp website

### PR DESCRIPTION
**Pull request by aakahlow**

### CODE REVIEW

File: modules/Using gem5/_posts/2000-05-01-models-cpu.md [hunk 1]
```diff
@@ -6,6 +6,71 @@ livestream_code: cDv-g-c0XCY
...
+- Using gem5 CPU models
+- Understanding the CPU model differences via an experiment
+
+**Note:** This document is meant to complement the slides used in the bootcamp and will refer to the slides for detailed instructions on how to run the experiments of this tutorial.
+
+## gem5 CPU Models
+
+Following picture provides an overview of the differences among gem5 CPU models:
+
+![Summary of gem5 CPU Models](../../../assets/img/gem5CPUs.png)
```
Comment (BobbyRBruce) [inline modules/Using gem5/_posts/2000-05-01-models-cpu.md:22]: I don't know if this kind of relatively linking will work, but it's more stable to do "/assets/img/gem5CPUs.png". This file (2000-05-01-models-cpu.md) could be moved at some point and doing it this way guarantees the link does not break.

File: modules/Using gem5/_posts/2000-05-01-models-cpu.md [hunk 2]
```diff
@@ -6,6 +6,71 @@ livestream_code: cDv-g-c0XCY
...
+### Memory Access Types in gem5:
+
+- Atomic: Useful for warming up micro-architectural structures, or fast-forwarding. An access completes within a sequence of nested calls.
+
+- Timing: An access is modeled using split transactions. This type of memory access models queuing delays and resource contention.
+An access request and response happens via separate functions.
+
+- Functional: Kind of a backdoor access to memory. Useful for debugging or whenever access to memory is needed without any impact on the simulated system (e.g., loading binaries into memory for SE mode).
+
+![Atomic and Timing Memory Accesses](../../../assets/img/memTypes.png)
```
Comment (BobbyRBruce) [inline modules/Using gem5/_posts/2000-05-01-models-cpu.md:34]: Ditto

**Review by BobbyRBruce — CHANGES_REQUESTED** [review]
I have two tiny comments. Not big at all. All the rest looks good to me

File: modules/Using gem5/_posts/2000-05-01-models-cpu.md [hunk 3]
```diff
@@ -6,6 +6,71 @@ livestream_code: cDv-g-c0XCY
...
+
+This CPU model is used for native execution on x86 and ARM platforms and can be used for fast-forwarding or functional testing.
+This CPU model uses the KVM (kernel virtual machine) support on the host to run the simulated guest on directly on the host.
+It requires the guest and host ISA to be the same.
+
+## Using the CPU Models
+
+This exercise is meant to cover the use of some of the gem5 CPU models to understand their differences.
+
+The location of the material (scripts/code) used for this exercise can be found in this [README](../../../materials/using-gem5/05-cpu-models/README.md).
```
[the remark this answers is not in the document]
Comment (aakahlow) [reply modules/Using gem5/_posts/2000-05-01-models-cpu.md]: Fixed.

File: modules/Using gem5/_posts/2000-05-01-models-cpu.md (hunk 1 above)
Comment (aakahlow) [reply modules/Using gem5/_posts/2000-05-01-models-cpu.md]: Fixed.

File: modules/Using gem5/_posts/2000-05-01-models-cpu.md (hunk 2 above)
Comment (aakahlow) [reply modules/Using gem5/_posts/2000-05-01-models-cpu.md]: Fixed.

**Review by BobbyRBruce — APPROVED** [review]

**Pull request merged**